### PR TITLE
IAA Headsets

### PR DESCRIPTION
**Pull request by Kyep**
🆑 Kyep
tweak: IAA radio headsets can now access med/eng/sci/supply/service channels in a similar manner to NTR headsets. They do not get command radio access.
/🆑

Details:
- IAAs now start with IAA headsets.
- IAA headsets are like NT Rep headsets, except that they don't get access to Command radio. 
- Like NTR headsets, they start out with only one channel enabled (Security), but can manually enable the rest of the channels they have access to.
- Like Sec headsets, including the headsets they already have, these headsets have flashbang protection.
- This PR was based on a suggestion by Streaky. The logic was that since IAAs are in theory meant to care about more than just Security, they should have access to more than just Security radio. **I'm not really pushing the idea, I'm mostly putting it up here so Streaky's suggestion can get seriously discussed.**

### CODE REVIEW

**Comment by StreakyHaddock** [comment]
IAAs are supposed to make sure departments communicate effectively, and that they're all following SOP and such. If science haven't had any minerals in the past 60 minutes, it's the IA who goes in and says "yeah, the miners are all dead." If Medical has had no chems made for the past half an hour because the chemists are fucking off, and security just arent responding/listening to the CMO's calls to help in their demotion, it's an IAA who grabs an officer and points him at the gas-masked labcoat wearing asshat with a syringe gun.

Inevitably the fear will be that their radio access will be misused, which I accept is possible. However, those that do misused radio are either gonna be demoted, dunked into a disposal bin without a headset, or ahelped. In addition, the hope is that this combined with the access increase will make the IAA Agent's job less tied to the brig and the security department, and draws people in who are interested in acting like regulatory bodies in a way that the NT REp and Magistrate aren't allowed to do. IA Agent can effectively function as an in-character, low-intensity mentor for the officer who keeps shooting the spider with disablers or a doctor who doesn't know how to do surgery.

tldr; allows them to do the "interdepartmental communication" part of their job more effetively, makes the job more interesting and open, helps them investigate, might draw enthusiastic players to a notoriously thankless job.

**Comment by Kyep** [comment]
Worth mentioning: their extra radio access has to be enabled at shift start (much like Captain, NTR, etc headsets). So, good IAA players are much more likely to enable these channels than bad IAA players are. Bad IAA players might not even realize they exist.

**Comment by LightFire53** [comment]
@kyep so with the shift of IAA being focused towards SOP, will these changes be added to the magistrate as their "head" or will they receive a new "commanding officer"?

**Comment by SamHPurp** [comment]
Perhaps rather than give them all this access... give them an assortment of headset keys to individually pick from to insert into their headset... rather than everything all the time.

**Comment by Kyep** [comment]
@LightFire53 I am not changing their head in this PR, unless general consensus wants me to.

**Comment by Fox-McCloud** [comment]
I don't think this is a good idea, personally.

IAA's are there to gather info and solve the a conflict or SoP/Spacelaw issue; they're not meant to be snoopers on everyone's business, as well; while it could be argued that the radio headsets will make them more "efficient" and grant additional context, I fear this is going to give them too much incursion into the goings on of specific departments.

This doesn't just make their department more efficient, it makes all of security more efficient since they can (and will) feed that info to the rest of sec (orrr sec will raid/ask for their stuff to make things "more efficient").

I'd rather not add additional levels of incursion into the department goings ons by security.

**Comment by Spartan6** [comment]
IAAs feel like they’re getting incredibly loot filled right now.

I could murder an IAA and get most comms and station access?

**Comment by granodd** [comment]
Well I mean, what better way to collect information than from the mouths of the department itself? If a department's doing something they're not supposed to do and talking about it in their coms, then it's good to have the IAA able to catch it. The NT Rep and Captain can already do this, though the NT Rep is more focused towards the heads of staff than they are the entire department, as that's more IAA's field.

**Comment by Ralta** [comment]
I have really mixed feelings on this. On the one hand, it'd be a great way to facilitate communication between departments and ensure they're working according to SOP.

On the other, we all know abuse is possible and as @Spartan6 has said, they become a bit of a loot pinata. 

Overall, I'd like to see the change implemented because I think it'd make IAA's that much more useful to the station. But it's also something I think will need to be monitored and potentially reverted if it's not used correctly.

**Comment by Jountax** [comment]
What if there was an all department (but command) intercom in the IAA office? something like the DJ station.

**Comment by davidchan13** [comment]
This PR needs to add smacking a round start IAA with a bat or metal rod causes candy to spill out instead of blood.

**Comment by StreakyHaddock** [comment]
Probably because sitting in an office isn't fun, and intercoms have massively more grief potential than headsets.

**Comment by Quantum-M** [comment]
The IAA has a flash and access to the security channel. If they were ended going into maintenance/a isolated place without an escort, then that's on them.

If there is a concern about IAAs relaying too much information to security, then maybe you could have them only use headset channels when they are investigating a department/a specific person from a department, and then disable said channel once they are done with the investigation.

**Comment by deep-fried-faceplant** [comment]
An agent's most important tool is good communication, both to give and receive information.  A good agent will make frequent use of their PDA's messenger in order to get in direct contact with witnesses, complainants, department heads and officials. While preforming routine investigations of security's arrests, access to security comms lets agents figure out things like who arrested a prisoner, when a prisoner is coming into the brig, what a prisoner's charge is, and whatever information they can squeeze out of security. Following this logic, one could assume that by giving agents departmental comms access, an agent could more efficiently solve departmental issues. However, this assumption relies on two other assumptions: that the agent knows how to properly utilize these channels of communication.

As it stands, the primary source of incompetence from the legal office is agents who attempt to make a case out of nothing, with a gung-ho attitude and a refusal to say no. These nitpicker agents will go to great lengths to make themselves relevant when they're not needed, from constantly interjecting during processing, to dragging prisoners off into interrogations, to yelling at any security officer they can point a finger at. In combination with basic access to all departments, access to all departmental radio channels will result in agents nitpicking other departments in order to find a case. Granted, an agent should be proactive and seek out cases, especially when they're not immediately apparent or the afflicted party doesn't report the incident. But unfortunately, we don't train agents to know when they should dive in and when they should pull back. The only means of quality control is admin intervention, which should be restricted to cases of extreme incompetence that border on malice. There will always be new IAA players, and from those new players, there will always be nitpickers. While experience can teach these nitpickers how to properly communicate while preforming investigations, the ever-present noobies will be there to shout nonsense at people trying to do their jobs. And while this dynamic does currently exist with the security radio channel and security access, it's a necessary evil so that good agents can quickly work on investigating arrests, where time is of the essence because of cell timers and security's constant workflow. Expanding it out to the entire crew can only end in disaster, or at least, a lot of people angry at busybody agents.

In short, I feel like the issue here is the overwhelming majority of agents who do not know how to use their current communication tools to their fullest potential, as well as the constant stream of gung-ho agents who are more likely to abuse the tools given to them. I'm personally split on the matter, because I know that I would do a good job with these tools, but I'm also a fairly experienced agent, so I don't very accurately represent the average IAA. This change feels like a bad move to make now, but not one that's impossible to revisit at a later date. I know that the admin team wants to crack down on rule 5 with IAAs, but until the bar gets raised to a point where we can trust the average IAA to not be an inefficient nuisance, agents should make do with what they've got.

As for Fox's argument that this change would make *security* more efficient: This change doesn't affect security. It's affecting IAAs, who do have strong ties with security, but in the antag vs. security fight, the only way IAAs would help security is by snooping on other department's radio chatter. If antagonists are openly talking about antagonistic things over departmental radio, then either the AI can relay it to security, or the antagonists are strong and confident enough that they can talk over the radio without fear of retribution.

**Comment by deep-fried-faceplant** [comment]
One last note: This all boils down to whether or not the benefits outweigh the drawbacks. This change will make good IAAs better, but bad IAAs worse. Note that bad IAAs are not necessarily breaking any rules, because we don't ban people for being bad at their job, only for failing to even try/acting with malice. While bad agents will eventually get weeded out, that's a lawn that requires constant maintenance by players and admins alike. Again, I'm fairly split on this, but to make a safe choice, my recommendation is to hold off on this change.

**Comment by Jountax** [comment]
@deep-fried-faceplant I feel the channels being off by default as in this PR will probably help to mitigate busybody newbies because they simply don't know they have the channels, and if they do, don't know how to turn them on.

**Comment by deep-fried-faceplant** [comment]
@Jountax Players who are new to IAA work are not necessarily new to the game overall. Experienced ss13 players who have no experience as an IAA will still fall down the same pitfalls that most players do when trying to get the hang of the legal office. Furthermore, I've noticed that these busy-bodies are typically semi-regular players, past the point of being a noob but not beyond the point of knowing how to correctly apply context to cases.

**Comment by Aurorablade** [comment]
I don't think this is a good idea to give them more then sec access.

**Comment by Varghh** [comment]
If killing pete the greytide IAA running around maints was already a good idea has antag because of sec if and hudglasses adding those radios make them just too damn juicy. I don't think this is a good idea.

**Comment by deep-fried-faceplant** [comment]
After thinking about it, I like @Jountax idea for a wall mounted radio. If people use it to grief, then we can more easily remove that player anyways. Spam is relatively easy to detect, given that it's a method of griefing meant to attract attention. If we're worried about agents misusing the intercom, we could limit it to only one department at a time, as well as including a note in the office about not abusing the intercom. Yes, it limits the agent to the office, but I disagree with Streaky and say that staying in the office can be fun. I always found ways to entertain myself without constantly running around, including shooting the breeze with the other agent, casual RP that was fun and contributed to the atmosphere.

However, I still don't like how much versatile power all-dept comms gives to agents. I would rather see something like @Quantum-M idea, that only one department is allowed to be snooped on at a time. Perhaps if a department fails inspection or if it's departmental SOP was broken, then an agent can obtain access to their comms from the NTR or Captain.

**Comment by Necaladun** [comment]
With the barrier to entry being so low, and IAA in general being...fairly awful, I can't really get behind this. IAA need a serious overhaul as it is, and I think this would just complicate it more.
